components/tutorial_tour_tip/tutorial_tour_tip.constant.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// This needs to replaced with the actual tour data based on product
+
+export const FINISHED = 999;
+
+export const AutoTourStatus = {
+    ENABLED: 0,
+    DISABLED: 1,
+};
+
+const AutoStatusSuffix = '_at_status';
+
+export const TutorialTourCategories: Record<string, string> = {
+    ADMIN_ON_BOARDING: 'admin_on_boarding',
+    CRT_TUTORIAL_STEP: 'crt_tutorial_step',
+    CRT_THREAD_PANE_STEP: 'crt_thread_pane_step',
+    TUTORIAL_STEP: 'tutorial_step',
+};
+
+export const AdminOnBoardingTourSteps = {
```

Comment:
I got it but I feel instead of writing down the logic to figure out if the user is admin or not and then skipping the step it would be a lot easier to maintain to list on frontend mostly array with the same value for step. Will push commit to show you what I mean.

Reply:
@neallred will take care of this in https://github.com/mattermost/mattermost-webapp/pull/9734 as current is only about New component these steps were just demo for other products

Reply:
Open to different implementations, and to resolving whatever issues there may be in that PR to keep this PR moving forward.

Like you say, the approach I was thinking of was to have a single tour and have a bunch of checks for whether the user is an admin and should see the step or not.

If I understand now, I think you are suggesting having two separate tours, one for admins, and one for non admins. That might be a better solution if the set of tips we are showing to each different user role differs greatly. If we do go that route, I think there's some things we will need to watch out for:
* Admin is a mutable role (even if usually doesn't change) on a more permanent user.
* Because admin status can change, I think we would want to prevent re-showing tours to users when they change admin status. I haven't thought of a clean way to do that.
    * Storing progress under the same preference entry for admin vs non-admin means needing to somehow know to change between the set of numbers when user admin status changes.
    * Storing the state of each tour (admin tour vs non admin tour) as its own separate entry in the preferences table means needing to advance *both* tours statuses as appropriate, so that when admin status changes they see the right tip.
    * Maybe this edge case isn't important enough and we're ok with them reseeing or skipping tips when they change admin status, or not seeing any tips ever (e.g. start as non-admin at step `-1`, get made an admin and never sees any admin tips because there is no corresponding `-1` for admins yet)
* Does the set of tour tips to be shown change for first admin vs other admins? If so, we're kind of back in the same spot of "make a separate tour list for first admin vs other admins" and "have only one tour set, but add conditional logic in the front end about which tips should be shown or skipped."